bigbluebutton-tests/playwright/chat/characterLimit.js

```diff
@@ -0,0 +1,28 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const { openChat } = require('./util');
+const e = require('../core/elements');
+
+class CharacterLimit extends Page {
+  constructor(browser, page) {
+    super(browser, page);
+  }
+
+  async test() {
+    await openChat(this.page);
+    const messageLocator = this.page.locator(e.chatUserMessageText);
+
+    await this.type(e.chatBox, e.longMessage5000);
+    await this.waitAndClick(e.sendButton);
+    await this.page.waitForSelector(e.chatUserMessageText);
+    await expect(messageLocator).toHaveCount(1);
+
+    await this.type(e.chatBox, e.longMessage5001);
+    await this.page.waitForSelector(e.typingIndicator);
```

Comment:
I didn't find the element with `data-test="typingIndicator"`. I think the best place to put it is in this span:
https://github.com/bigbluebutton/bigbluebutton/blob/9696517750851c82e359e91ecf455e2e7acce31f/bigbluebutton-html5/imports/ui/components/chat/message-form/typing-indicator/component.jsx#L45-L50